go/config/host_config.go

```diff
@@ -37,10 +37,8 @@ type HostInputConfig struct {
 	P2PBindAddress string
 	// P2PPublicAddress is the advertised P2P server address
 	P2PPublicAddress string
-	// The host of the connected L1 node
-	L1NodeHost string
-	// The websocket port of the connected L1 node
-	L1NodeWebsocketPort uint
+	// Websocket RPC address for interactions with the L1
+	L1WebsocketRPCAddress string
```

Comment:
```suggestion
	// L1WebsocketURL is the RPC address for interactions with the L1
	L1WebsocketURL string
```

Keeping the same name for consistency